### PR TITLE
SAURABH| fix url logo and themes

### PR DESCRIPTION
**Pull request by Saurabh186**

### CODE REVIEW

File: src/components/collectionVersions/collectionParentPages.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react'
-import { Card, Dropdown, Accordion, DropdownButton, Button } from 'react-bootstrap'
+import React, { Component, useState } from 'react'
```
Comment (idris-bohra) [inline src/components/collectionVersions/collectionParentPages.jsx:1]: why useState in react classbased component

File: src/components/collectionVersions/collectionParentPages.jsx [hunk 2]
```diff
@@ -468,19 +474,30 @@ class CollectionParentPages extends Component {
...
     let isUserOnPublishedPage = isOnPublishedPage()
     const expanded = this.props?.clientData?.[pageId]?.isExpanded ?? isUserOnPublishedPage
     const publishData = this.props.modals.publishData
     const rootId = pageId
     const isSelected = isUserOnPublishedPage && sessionStorage.getItem('currentPublishIdToShow') === pageId ? 'selected' : (isDashboardRoute && this.props.match.params.pageId === pageId ? 'selected' : '')
+    let idToRender = sessionStorage.getItem(SESSION_STORAGE_KEY.CURRENT_PUBLISH_ID_SHOW) || this.state.idToRenderState;
+    let collectionId = this.props?.pages?.[idToRender]?.collectionId ?? null
+    var collectionTheme = this.props.collections[collectionId]?.theme
+    const dynamicColor = hexToRgb(collectionTheme, 0.15);
+    const staticColor = '#bdbdbd';
```
Comment (idris-bohra) [inline src/components/collectionVersions/collectionParentPages.jsx:487]: Add this as a enum and also make the variable of this color in scss

File: src/components/endpoints/displayEndpoint.jsx [hunk 3]
```diff
@@ -418,6 +420,20 @@ class DisplayEndpoint extends Component {
       const { ipcRenderer } = window.require('electron')
       ipcRenderer.on('ENDPOINT_SHORTCUTS_CHANNEL', this.handleShortcuts)
     }
+    const dynamicColor = hexToRgb(this.props.publicCollectionTheme, 0.02);
+    const staticColor = '#f5f5f5';
```
Comment (idris-bohra) [inline src/components/endpoints/displayEndpoint.jsx:424]: Also make this color in scss and enums file

File: src/components/endpoints/displayEndpoint.jsx [hunk 4]
```diff
@@ -2826,7 +2842,7 @@ class DisplayEndpoint extends Component {
     }
 }
 
-  render() {
+  render(props) {
```
Comment (idris-bohra) [inline src/components/endpoints/displayEndpoint.jsx:2845]: remove this if not used and if used then rename this

File: src/components/endpoints/displayResponse.jsx [hunk 5]
```diff
@@ -115,9 +115,19 @@ class DisplayResponse extends Component {
   }
 
   componentDidMount() {
-    if (!this.state.theme) {
-      this.setState({ theme: this.props.publicCollectionTheme })
-    }
+    const dynamicColor = hexToRgb(this.props.publicCollectionTheme, 0.02);
+    const staticColor = '#fafafa';
```
Comment (idris-bohra) [inline src/components/endpoints/displayResponse.jsx:119]: Add the variable Enum and scss

File: src/components/endpoints/endpoints.jsx [hunk 6]
```diff
@@ -339,6 +344,18 @@ class Endpoints extends Component {
     const idToCheck = this.props.location.pathname.split('/')[4] === 'endpoint' ? this.props.location.pathname.split('/')[5] : null
     let isUserOnPublishedPage = isOnPublishedPage()
     const isSelected = isUserOnPublishedPage && sessionStorage.getItem('currentPublishIdToShow') === endpointId ? 'selected' : (isDashboardRoute && this.props.match.params.endpointId === endpointId ? 'selected' : '')
+    let idToRender = sessionStorage.getItem(SESSION_STORAGE_KEY.CURRENT_PUBLISH_ID_SHOW) || this.state.idToRenderState;
+    let collectionId = this.props?.pages?.[idToRender]?.collectionId ?? null
+    var collectionTheme = this.props.collections[collectionId]?.theme
+    const dynamicColor = hexToRgb(collectionTheme, 0.15);
+    const staticColor = '#bdbdbd';
```
Comment (idris-bohra) [inline src/components/endpoints/endpoints.jsx:351]: ??

File: src/components/endpoints/endpoints.scss [hunk 7]
```diff
@@ -1257,10 +1257,11 @@ table#custom-generic-table {
   width: 100%;
 
   nav.nav.nav-tabs {
-    border: 1px solid #e0e0e0;
+    border: 1px solid #0000001f;
```
Comment (idris-bohra) [inline src/components/endpoints/endpoints.scss:1260]: 0000001f make this as variable in scss

File: src/components/endpoints/genericTable.jsx [hunk 8]
```diff
@@ -38,7 +38,19 @@ class GenericTable extends Component {
   }
 
   componentDidMount() {
-    this.setState({ optionalParams: false, theme: this.props.publicCollectionTheme })
+    const dynamicColor = hexToRgb(this.props.publicCollectionTheme, 0.02);
+    const staticColor = '#fafafa';
```
Comment (idris-bohra) [inline src/components/endpoints/genericTable.jsx:42]: Enum and scss variable

File: src/components/endpoints/publicBodyContainer.jsx [hunk 9]
```diff
@@ -45,6 +48,26 @@ class PublicBodyContainer extends Component {
       urlencoded: this.props?.body?.[bodyTypesEnums['application/x-www-form-urlencoded']] || [{ checked: 'notApplicable', key: '', value: '', description: '', type: 'text' }],
     }
     this.setState({ data });
+    this.updateBackgroundStyle();
+
+  } updateBackgroundStyle() {
+    const { publicCollectionTheme } = this.state.theme;
+    const dynamicColor = hexToRgb(publicCollectionTheme, 0.02);
+    const staticColor = '#fafafa';
```
Comment (idris-bohra) [inline src/components/endpoints/publicBodyContainer.jsx:56]: Enum and scss var

File: src/components/endpoints/publicSampleResponse.jsx [hunk 10]
```diff
@@ -7,10 +7,37 @@ import { Style } from 'react-style-tag'
...
+      },
+    };
   }
+  componentDidMount() {
+    this.updateBackgroundStyle();
+  }
+  updateBackgroundStyle() {
+    const { publicCollectionTheme } = this.state.theme;
+    const dynamicColor = hexToRgb(publicCollectionTheme, 0.02);
+    const staticColor = '#fafafa';
```
Comment (idris-bohra) [inline src/components/endpoints/publicSampleResponse.jsx:25]: Enum and scss var

File: src/components/main/main.scss [hunk 11]
```diff
@@ -1158,7 +1158,7 @@ body .form-group {
 
 body .form-control {
   height: 36px;
-  border: 1px solid $outline-color;
+  border: 1px solid #0000001f;
```
Comment (idris-bohra) [inline src/components/main/main.scss:1161]: Enum and scss var

File: src/components/main/sidebar.scss [hunk 12]
```diff
@@ -55,7 +55,7 @@
 
 .Resizer {
   width: 1px;
-  background: #e6dbd4;
+  background: #0000001f;
```
Comment (idris-bohra) [inline src/components/main/sidebar.scss:58]: Enum and scss var

File: src/components/main/sidebar.scss [hunk 13]
```diff
@@ -894,8 +887,8 @@ input[type='text']:focus {
 }
 
 .search-container {
-  border: 1px solid rgba(0, 0, 0, 0.104);
-  padding: 4px;
+  border: 1px solid #0000001f;
```
Comment (idris-bohra) [inline src/components/main/sidebar.scss:890]: Enum and scss var

File: src/components/main/sidebar.scss [hunk 14]
```diff
@@ -972,7 +964,7 @@ input[type='text']:focus {
   position: fixed;
   bottom: 0;
   left: 0;
-  border-top: 1px solid rgb(221, 205, 205);
+  border-top: 1px solid #0000001f;
```
Comment (idris-bohra) [inline src/components/main/sidebar.scss:967]: Enum and scss var

**Review by idris-bohra — CHANGES_REQUESTED** [review]
Hover on subPage and give more padding to the subpage and parent page same as the collection.

Restore the hover of expand and collapse icon.

File: src/components/collectionVersions/collectionParentPages.jsx (hunk 1 above)
Comment (Saurabh186) [reply src/components/collectionVersions/collectionParentPages.jsx:1]: done

File: src/components/collectionVersions/collectionParentPages.jsx (hunk 2 above)
Comment (Saurabh186) [reply src/components/collectionVersions/collectionParentPages.jsx:487]: done

File: src/components/endpoints/displayEndpoint.jsx (hunk 3 above)
Comment (Saurabh186) [reply src/components/endpoints/displayEndpoint.jsx:424]: done

File: src/components/endpoints/displayEndpoint.jsx (hunk 4 above)
Comment (Saurabh186) [reply src/components/endpoints/displayEndpoint.jsx:2845]: dome

File: src/components/endpoints/displayResponse.jsx (hunk 5 above)
Comment (Saurabh186) [reply src/components/endpoints/displayResponse.jsx:119]: done

File: src/components/endpoints/endpoints.jsx (hunk 6 above)
Comment (Saurabh186) [reply src/components/endpoints/endpoints.jsx:351]: done

File: src/components/endpoints/endpoints.scss (hunk 7 above)
Comment (Saurabh186) [reply src/components/endpoints/endpoints.scss:1260]: done

File: src/components/endpoints/genericTable.jsx (hunk 8 above)
Comment (Saurabh186) [reply src/components/endpoints/genericTable.jsx:42]: done

File: src/components/endpoints/publicBodyContainer.jsx (hunk 9 above)
Comment (Saurabh186) [reply src/components/endpoints/publicBodyContainer.jsx:56]: done

File: src/components/endpoints/publicSampleResponse.jsx (hunk 10 above)
Comment (Saurabh186) [reply src/components/endpoints/publicSampleResponse.jsx:25]: done

File: src/components/main/main.scss (hunk 11 above)
Comment (Saurabh186) [reply src/components/main/main.scss:1161]: done

File: src/components/main/sidebar.scss (hunk 12 above)
Comment (Saurabh186) [reply src/components/main/sidebar.scss:58]: done

File: src/components/main/sidebar.scss (hunk 13 above)
Comment (Saurabh186) [reply src/components/main/sidebar.scss:890]: done

File: src/components/main/sidebar.scss (hunk 14 above)
Comment (Saurabh186) [reply src/components/main/sidebar.scss:967]: done

**Review by idris-bohra — CHANGES_REQUESTED** [review]

**Review by idris-bohra — APPROVED** [review]

**Pull request merged**